crates/protogen/src/metastore/types/service.rs

```diff
@@ -691,20 +735,3 @@ impl From<UpdateDeploymentStorage> for service::UpdateDeploymentStorage {
         }
     }
 }
-
-#[cfg(test)]
-mod tests {
```

Comment:
can you add this back

Reply:
no, we can't proptest anymore as `Signature` isn't compatible with `Arbitrary`.